feat: remove useless deployed spore contract

ashuralyk opened the pull request:
# Description
split `toml` directory into `mainnet` and `testnet` two sub-directories, and then, consume an useless deployed Spore contract cell and withdraw its locked capacity.

besides, writing a toolchain to automatically manage contract deployment is not easy, so I've suggested we just regulate a principle of deployment as the method.

cc @Flouse @ShookLyngs 

# Related Issues
- [x] #49

joii2020 commented:
I found that before, the `tests` in ./Cargo.toml were moved from `exclude` to `members`. But this will lead to: every time the dependencies of tests are modified, it may affect the compilation of on-chain script.

ashuralyk commented:
> I found that before, the `tests` in ./Cargo.toml were moved from `exclude` to `members`. But this will lead to: every time the dependencies of tests are modified, it may affect the compilation of on-chain script.

rebased and fixed
